### DISCUSSION TITLE
showing screen shots on web site?

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#1191] [Ideas]
For the web site... showing screen shots?  A "proof of concept":

```
#!/bin/sh

pushd /tmp
wget https://github.com/orgs/Interlisp/discussions/1097

grep --only-matching "https://user-images[a-z0-9/.-]*\.png" 1097 | uniq > /tmp/screenshots
```

will download a list of images... some of which at least are full-size images of Medley screens.

so the idea is to periodically update a list of images that can be displayed using a javascript slide show, on the web site... perhaps with some manual curation? (annotate the images wanted?)

https://www.w3schools.com/howto/howto_js_slideshow.asp

As it stands. the screenshots are a little hard to find and look through, so this might make it more useful.
I couldn't find any way to get the Discussion number by lookup.

**Comment by pamoroso** [comment]
It's a great idea. As for the discussion numbers, maybe the GitHub API provides access to such metadata.
